c/build_all/linux/setup.sh

```diff
@@ -12,7 +12,7 @@ repo_name_from_uri()
 }
 
 scriptdir=$(cd "$(dirname "$0")" && pwd)
-deps="curl libcurl4-openssl-dev uuid-dev uuid g++ make cmake git"
+deps="curl libcurl4-openssl-dev uuid-dev uuid g++ make cmake unzip openjdk-7-jre"
```

Comment:
I asked because I know that Proton-C builds on windows without Java; I wasn't sure about Linux.  However, I just spun up a fresh Ubuntu VM (14.04 LTS) and confirmed that it didn't have Java installed.  Then I ran the script.  It successfully built and installed Proton-C without errors.  Do you have more information about the errors you were seeing?